nf_core/launch.py

```diff
@@ -703,6 +704,7 @@ def build_command(self):
                 with open(self.params_out, "w") as fp:
                     json.dump(self.schema_obj.input_params, fp, indent=4)
                     fp.write("\n")
+                run_prettier_on_file(self.params_out)
```

Comment:
Isn't this formatting a params input file? I don't think this is necessary? Although it doesn't do any harm I guess?

Reply:
Yes, probably not necessary, but I thought that it's better to prevent possible linting fails, now that we are formatting everything 🤔